components/NotificationBar.js

```diff
@@ -117,6 +150,20 @@ class NotificationBar extends React.Component {
           <div className={`NotificationLine ${status}`}>
             <h1>{title}</h1>
             <p className="description">{description}</p>
+
+            {isHostAdmin && (
+              <AcceptRejectBanner>
+                <AcceptRejectButtons collective={collective} host={host} />
+                {this.state.showRejectionModal && (
+                  <AppRejectionReasonModal
```

Comment:
`AcceptRejectButtons` already implements the modal. Why would we need to display it here too? Where is `showRejectionModal` set?

Reply:
Done